aspnetcore.yaml

```diff
@@ -2,9 +2,9 @@ steps:
 - name: 'gcr.io/gcp-runtimes/aspnetcorebuild:latest'
   args:
     - '--version-map'
-    - '1.0.8=gcr.io/google-appengine/aspnetcore:1.0.8'
-    - '1.1.5=gcr.io/google-appengine/aspnetcore:1.1.5'
-    - '2.0.4=gcr.io/google-appengine/aspnetcore:2.0.4'
+    - '1.0.9=gcr.io/google-appengine/aspnetcore:1.0.9'
```

Comment:
Are only the newest minor version of each major version tested?

Reply:
You raise a really good point. Up until know that's what I have been doing, maybe we should have tests for the previous ones? I have also opened #86 to track cleaning up old images, which can make this work not needed.